tools/recetox_xmsannotator/.shed.yml

```diff
@@ -0,0 +1,16 @@
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/recetox-xmsannotator"
+homepage_url: "https://github.com/RECETOX/recetox-xMSannotator"
+categories:
+  - Metabolomics
+repositories:
+  recetox_xmsannotator_simple:
+    description: "Annotate peak table with metabolites"
+    include:
+      - recetox_xmsannotator_simple.xml
+      - macros.xml
+  recetox_xmsannotator_advanced:
+    description: "Annotate peak table with metabolites"

```

Comment:
Could be made more detailed to "Annotate peak table using RT, m/z and mass defect based clustering, isotopic patterns, pathways and chemical score computation."

Reply:
```suggestion
    description: "Annotate peak table using RT, m/z and mass defect based clustering, isotopic patterns, pathways and chemical score computation"

```